AFQ/tasks/segmentation.py

```diff
@@ -214,6 +214,46 @@ def export_sl_counts(data_imap,
     return counts_df, dict(sources=bundles_files)
 
 
+@pimms.calc("sl_lengths")
+@as_file('_sl_lengths.csv', include_track=True, include_seg=True)
+def export_sl_lengths(data_imap,
```

Comment:
Just one more nit here: do you want the name of the function to reflect the fact that these are *median* counts? Right now, it gives the impression that these are somehow "raw" counts.